### PR TITLE
Fable forms safe v5

### PR DESCRIPTION
**Pull request by Larocceau**
* Make compatible with Vite
* Simplify
* Fix code snippets
* make code snippets coherent with V5 template

### CODE REVIEW

File: docs/recipes/client-server/fable.forms.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+```
+
+## Register styles
+
+1. Rename `src/Client/Index.css` to `Index.scss`
+
+   2. Update the import in `App.fs`
+
+       === "Code"    
+           ```.fs title="App.fs"
```
Comment (theprash) [inline docs/recipes/client-server/fable.forms.md:31]:
```suggestion
           ```.fsharp title="App.fs"
```

Comment (theprash) [reply docs/recipes/client-server/fable.forms.md:31]: This fixes syntax highlighting.

File: docs/recipes/client-server/fable.forms.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+3. Import bulma and fable-form-simple in `Index.scss`
+
+    === "Code"
+        ``` .scss title="Index.scss"
+        @import "~bulma";
+        @import "~fable-form-simple-bulma";
+        ...
+        ```
+    === "Diff"
```
Comment (theprash) [inline docs/recipes/client-server/fable.forms.md:52]: I don't think we need both code and diff when we're only adding lines, so could you remove the diff and the "code" heading?

File: docs/recipes/client-server/fable.forms.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+1. Open the newly added namespaces:
+
+    === "Code"
+        ``` { .fsharp title="Index.fs" }
+        open Fable.Form.Simple
+        open Fable.Form.Simple.Bulma
+        ```
+
+    === "Diff"
```
Comment (theprash) [inline docs/recipes/client-server/fable.forms.md:79]: Similar to above, the diff isn't really needed.

**Review by theprash — CHANGES_REQUESTED** [review]

File: docs/recipes/client-server/fable.forms.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+        + importSideEffects "./index.scss"
+        ...
+        ```
+
+3. Import bulma and fable-form-simple in `Index.scss`
+
+    ``` .scss title="Index.scss"
+    @import "bulma/bulma.sass";
+    @import "fable-form-simple-bulma/index.scss";
+    ...
```
Comment (Larocceau) [inline docs/recipes/client-server/fable.forms.md:49]: I realized when implementing your feedback that I made a mistake here... Vite does not deal well with the  "~" imports, so we need to use this notation

**Review by theprash — APPROVED** [review]

**Pull request merged**